.travis.yml

```diff
@@ -25,4 +25,6 @@ notifications:
       - almartin@gmail.com
       - chaid@kippchicago.org 
     on_success: always
-    on_failure: always
\ No newline at end of file
+    on_failure: always
+  
+  hipchat: 1f5638b987cc31020439c32e78554e@kippdata
```

Comment:
that was going to let travis push build status notifications into our hipchat room, but the rec is you should encrypt your API key, and to do that I need command line travis, and to do that I fell into windows cygwin dependency hell that i still haven't fully gotten out of.